src/renderers/WebGLRenderer.js

```diff
@@ -1108,6 +1108,14 @@ function WebGLRenderer( parameters ) {
 		_currentMaterialId = - 1;
 		_currentCamera = null;
 
+		// onUpdate
+
+		scene.traverse( function( node ) {
+
+			node.onUpdate( _this, scene, camera );
```

Comment:
Ah... Good point... Hmm... So tempting to just putting it in `updateMatrixWorld()`...